proxy/coco/src/peer.rs

```diff
@@ -80,7 +80,11 @@ impl Api {
             run_loop.await;
         });
 
-        // Register the rad:// transport protocol
+        // Turns out we don't need to call the git commands from the command line.
+        // All we need is to register the git transport and this is the same
+        // thing. This means we can use git2 to execute actions. So instead of
+        // implementing the checkout via git clone as a Command, we use
+        // [`git2::RepoBuilder`].
```

Comment:
This comment seems out of place. I get that it's trying to explain the history of a misconception we had. I think it would make more sense to explain what it buys us, for example:

This registers the `rad://` transport protocol so that we can perform git actions between the Monorepo and a working copy. This means that we can easily use the `git2` library to perform these actions and be sure that the `signer` is used to sign any git artifacts.